internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-basic-manifest.yml

```diff
@@ -725,6 +725,7 @@ Resources:
       LoadBalancerArn: !Ref PublicLoadBalancer
       Port: 443
       Protocol: HTTPS
+      SslPolicy: 'ELBSecurityPolicy-2016-08'
```

Comment:
Because you have added the template with ALB security policy, we probably don't need to add it to all of these other templates, especially the basic manifest one.

Reply:
Oh, nvm. I see-- this is the default. Because it's not a required field, can we just not include it if the user doesn't specify a non-default policy?

Reply:
sure I will changes accordingly